src/lib/components/Codelab/index.js

```diff
@@ -33,7 +33,7 @@ class Codelab extends BaseElement {
     // _isDesktop has no default value as it's only correctly set between connected/disconnected
     // callbacks via the MediaQueryList's listener.
 
-    this._mql = window.matchMedia('(min-width: 865px)');
+    this._mql = window.matchMedia('(min-width: 800px)');
```

Comment:
I think this new breakpoint is too small. at 830px codelab looks like this:
<img width="859" alt="Screenshot 2021-12-20 at 15 43 32" src="https://user-images.githubusercontent.com/1914261/146785384-6eb7d369-a510-4a04-a7cf-c742a4f181ab.png">

Reply:
I kept the 800px bp, but changed the size of the columns so it should look better at the smaller viewport

![image](https://user-images.githubusercontent.com/11811422/148603309-43a89344-735a-41ff-9d07-356c95c4c102.png)

Reply:
If you don't have the sidebar closed (which I didn't by default), it looks like this, which is not ideal because the column collapse button is covered by the "Remix to Edit" banner:

<img width="805" alt="Screen Shot 2022-01-07 at 1 07 09 PM" src="https://user-images.githubusercontent.com/326742/148607644-1de7878b-41f2-4690-9db3-9768069e14c5.png">